Documentation/contributing.rst

```diff
@@ -812,303 +960,370 @@ Have a question about how the tests work or want to chat more about improving th
-    cat /var/run/cilium/state/bpf_features.log
-    BPF/probes: CONFIG_CGROUP_BPF=y is not in kernel configuration
-    BPF/probes: CONFIG_LWTUNNEL_BPF=y is not in kernel configuration
-    HAVE_LPM_MAP_TYPE: Your kernel doesn't support LPM trie maps for BPF, thus disabling CIDR policies. Recommendation is to run 4.11+ kernels.
-    HAVE_LRU_MAP_TYPE: Your kernel doesn't support LRU maps for BPF, thus switching back to using hash table for the cilium connection tracker. Recommendation is to run 4.10+ kernels.
+#. All commits are signed off. See the section :ref:`dev_coo`.
 
-Current BPF map state for particular programs is held under ``/sys/fs/bpf/``,
-and the `bpf-map <https://github.com/cilium/bpf-map>`_ utility can be useful
-for debugging what is going on inside them, for example:
+#. Pick the appropriate milestone for which this PR is being targeted to, e.g.
+   ``1.1``, ``1.2``. This is in particular important in the time frame between
+   the feature freeze and final release date.
 
-.. code:: bash
+#. Pick the right release-note label
 
-    # ls /sys/fs/bpf/tc/globals/
-    cilium_calls_15124  cilium_calls_48896        cilium_ct4_global       cilium_lb4_rr_seq       cilium_lb6_services  cilium_policy_25729  cilium_policy_60670       cilium_proxy6
-    cilium_calls_25729  cilium_calls_60670        cilium_ct6_global       cilium_lb4_services     cilium_lxc           cilium_policy_3978   cilium_policy_reserved_1  cilium_reserved_policy
-    cilium_calls_3978   cilium_calls_netdev_ns_1  cilium_events           cilium_lb6_reverse_nat  cilium_policy        cilium_policy_4314   cilium_policy_reserved_2  cilium_tunnel_map
-    cilium_calls_4314   cilium_calls_overlay_2    cilium_lb4_reverse_nat  cilium_lb6_rr_seq       cilium_policy_15124  cilium_policy_48896  cilium_proxy4
-    # bpf-map info /sys/fs/bpf/tc/globals/cilium_policy_15124
-    Type:           Hash
-    Key size:       8
-    Value size:     24
-    Max entries:    1024
-    Flags:          0x0
-    # bpf-map dump /sys/fs/bpf/tc/globals/cilium_policy_15124
-    Key:
-    00000000  6a 01 00 00 82 23 06 00                           |j....#..|
-    Value:
-    00000000  01 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
-    00000010  00 00 00 00 00 00 00 00                           |........|
+   +--------------------------+---------------------------------------------------------------------------+
+   | Labels                   | When to set                                                               |
+   +==========================+===========================================================================+
+   | ``release-note/bug``     | This is a non-trivial bugfix                                              |
+   +--------------------------+---------------------------------------------------------------------------+
+   | ``release-note/major``   | This is a major feature addition                                          |
```

Comment:
Add an example of what is a major/minor feature addition

Reply:
added